docs/table/access_table.rst

```diff
@@ -680,8 +676,8 @@ expressions (see the warning below for caveats to this)::
 Bytestring columns in Python 3
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Prior to astropy 2.0, using bytestring columns (numpy ``'S'`` dtype) in Python
-3 was inconvenient because it was not possible to compare with the natural
+Starting with astropy 2.0, using bytestring columns (numpy ``'S'`` dtype) in Python
+3 is straightforward since they can be easily compared with the natural
```

Comment:
We're python3 only for a while now, so is this still necessary to refer back to the features of this python version?